app/lib/onboarding/follow_recommendation_category.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# Represents a grouping of suggested accounts that can be followed by new users on moth.social.
+# Categories are grouped by theme, eg.: tech, news, art, etc.
```

Comment:
does this theme show up in the app anywhere, or is it just for our own knowledge?

Reply:
it doesnt, actually I think @rileyhCode wanted to get it removed sinde its not gonna be used by the client at all. I'll confirm with him